be/src/common/config.h

```diff
@@ -178,10 +178,10 @@ CONF_Bool(serialize_batch, "false");
 CONF_mInt32(status_report_interval, "5");
 // Local directory to copy UDF libraries from HDFS into.
 CONF_String(local_library_dir, "${UDF_RUNTIME_DIR}");
-// Number of olap scanner thread pool size.
-CONF_mInt32(doris_scanner_thread_pool_thread_num, "48");
-// Number of olap scanner thread pool size.
-CONF_Int32(doris_scanner_thread_pool_queue_size, "102400");
+// Number of olap/external scanner thread pool size.
+CONF_mInt32(table_scanner_thread_pool_thread_num, "48");
```

Comment:
I think you'd better to keep old config to be compatible and introduce these two.
If some guys config these configs in the be.conf, we should take the config.
So better to do some compatible work like 
```

thread_num = config::table_scanner_thread_pool_thread_num;
if (thread_num == 48 && config::doris_scanner_thread_pool_thread_num != 48) {
    thread_num = config::doris_scanner_thread_pool_thread_num
}

```

Reply:
ok, let's keep config names instead of new configs